workspaces/meewee-time-registration/README.md

```diff
@@ -0,0 +1,10 @@
+# [Backstage](https://backstage.io)
```

Comment:
Can we update this with a proper title and perhaps links to the installation docs for each plugin?

Reply:
Done 👍